sklearn/utils/tests/test_utils.py

```diff
@@ -685,3 +688,22 @@ def test_to_object_array(sequence):
     assert isinstance(out, np.ndarray)
     assert out.dtype.kind == "O"
     assert out.ndim == 1
+
+
+@pytest.mark.parametrize("file_extension", ["pyx", "pxd"])
+def test_files_generation_from_templates(file_extension):
+    base_dir = base_dir = pathlib.Path(sklearn.__file__).parent.parent
+    template_filename = f"dummy.{file_extension}.tp"
+    template_file = base_dir / template_filename
+
+    f = open(template_file, "a")
+    f.close()
+
+    templates = [template_filename]
+    gen_from_templates(templates)
+
+    generated_file = base_dir / f"dummy.{file_extension}"
+    assert generated_file.exists()
+
+    template_file.unlink()
+    generated_file.unlink()
```

Comment:
@jjerphan Do you think testing this behavior is necessary?

@ogrisel and I were were thinking that it may be redundant to test if `Tempita` is working as expected.